nix/npm.nix

```diff
@@ -0,0 +1,38 @@
+{ system, dream2nix-lib, nix-filter, nodejs }:
+
+let
+  outputs = (dream2nix-lib.makeFlakeOutputs {
+    source = nix-filter.lib.filter {
+      root = ../.;
+      include = [ ../package-lock.json ../package.json ];
+    };
+
+    packageOverrides = {
+      webpack-cli = {
+        remove-webpack-check = {
+          patches = [ ./patches/remove-webpack-check.patch ];
+        };
+      };
+    };
+
+    inject = { acorn-import-assertions."1.8.0" = [[ "acorn" "8.7.1" ]]; };
```

Comment:
Curious, what is this?

Reply:
I think it's a dependency of webpack, I tried removing it but then it complained, we should add a comment